src/Common/MemoryWorker.cpp

```diff
@@ -236,7 +236,7 @@ MemoryWorker::MemoryWorker(uint64_t period_ms_, bool correct_tracker_, bool use_
             static constexpr uint64_t cgroups_memory_usage_tick_ms{50};
 
             const auto [cgroup_path, version] = getCgroupsPath();
-            LOG_INFO(
+            LOG_TRACE(
```

Comment:
It is really necessary? and the later mod

Reply:
The INFO-level logs would be printed directly to standard output. For example, when a user runs a Python script, they would first see a string of content like "Will create cgroup reader from..." before the query results are displayed. Therefore, the log level has been changed to TRACE.